CHANGELOG.rst

```diff
@@ -13,6 +13,7 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 * Experimental designs can be defined for an offer (`#263 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/263>` _)
 * New dropdown menu bar (`#490` <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/490>` _)
+* Jump back to maintain view after product creation/update (`#481` <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/481>` _)
```

Comment:
```suggestion

* Jump back to maintain view after product creation/update (`#481 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/481>` _)
```
I know we decided against review comments but I just noticed those small changes

Reply:
We can also fix the CL when we prepare the release, I guess we would still need to do that :)

Reply:
Since it is just 2 small typos, I thought I would simply do a suggestion here directly.